### PR TITLE
feat: added option to specify filname to pull-components command

### PR DESCRIPTION
**Pull request by mattcrn**
## Pull request type

Jira Link: -

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

* Run the command with the `-f` flag to specify an file identifier

```bash
node src/cli.js pull-components -f production --space 123456
``` 
The resulting file should be called `components.production.json` instead of `components.123456.json`

* Test if it works with all other options

## What is the new behavior?

- adds `-f` flag to specify the file identifier for the pull-component command
- this gives the user the option to have files saved not with space id as suffix but with a custom id i.e.: `components.production.json`
- This is helpful for keeping track of different versions of components, or for merging components from different spaces

## Other information

I created a similar PR last year for the old repo, I don't know what happened with it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "storyblok",
-  "version": "3.15.0",
+  "name": "towa-storyblok",
+  "version": "3.16.0",
```
Comment (ademarCardoso) [inline package.json]: Please man, revert these changes

**Review by ademarCardoso — APPROVED** [review]
It's workin really nice, thanks for the contribution, when you fix my comment we can merge 😄

**Comment by mattcrn** [comment]
Hey @ademarCardoso I have reverted the changes in the `package.json`
